java/src/jmri/Timebase.java

```diff
@@ -3,8 +3,11 @@
 import java.beans.PropertyChangeListener;
 import java.time.Instant;
 import java.util.Date;
+
 import javax.annotation.Nonnull;
 
+import jmri.time.TimeProvider;
```

Comment:
It might make sense to have the `TimeProvider` interface in the `jmri` package so we don't have to import from a sub package here.  Just `TimeProvider`, not the rest of the new `jmri.time` package.